config/default.schema.json

```diff
@@ -194,6 +205,7 @@
 	},
 	"required": [
 		"TEACHER_VISIBILITY_FOR_EXTERNAL_TEAM_INVITATION",
-		"COOKIE"
+		"COOKIE",
+		"STUDENT_TEAM_CREATION"
```

Comment:
if setting this required, which is good, the default opt-in has to be added into the default.json too.

Reply:
Done